ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml

```diff
@@ -50,7 +51,12 @@ StackView {
 
     // Network related properties:
     property var flatNetworks
-    readonly property int ownerTokenChainId: SQUtils.ModelUtils.get(root.tokensModel, "privilegesLevel", Constants.TokenPrivilegesLevel.Owner).chainId ?? 0
+    readonly property int ownerTokenChainId: {
```

Comment:
Why not using the `ModelEntry` to get the ownerToken data?

The disadvantage for the `ModelCount.count` dependency is that the bindings in the panel are triggered unnecessarily whenever a new token is minted, even if the owner token never changes.

Reply:
Yes, that's true. It's just the first step to eliminate all logical errors like using `get` instead of `getByKey`.

Reply:
In the end we have two conditions there and therefore it's not sth we could easily cover with `ModelEntry`. I'd leave it as it is for now as it's sufficient there.